packages/react-components/react-portal-compat/package.json

```diff
@@ -12,11 +12,11 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
+    "build": "just-scripts build --module esm,cjs,amd",
```

Comment:
this package should also be AMD since it is intended to be used with `react-portal-compat-context`